esx_service/vmci/vmci_client.c

```diff
@@ -183,13 +181,12 @@ vsock_init(be_sock_id *id, int cid, int port)
    int sock;     // socket id
 
    if ((af = vsock_get_family()) == -1) {
-      perror("Failed to get address family.");
-      return CONN_FAILED_VMCI_ADDRESS_FAMILY_GET;
+      errno = EAFNOSUPPORT;
```

Comment:
Actually i twas not set in client .It is a leftover from totally separate client/server files. 
I moved the vsock_get_family to the header fie (so it's shared) and dropped extra errno set in both client and server